tests/phpunit/tests/ajax/MediaEdit.php

```diff
@@ -67,6 +67,7 @@ public function testCropImageThumbnail() {
 	 */
 	public function testImageEditOverwriteConstant() {
 		define( 'IMAGE_EDIT_OVERWRITE', true );
+		$files_that_shouldnt_exist = array();
```

Comment:
this test was failing with `$files_that_shouldnt_exist` undefined below (https://github.com/adamsilverstein/wordpress-develop-fork/pull/35/files#diff-7e3d8ea591299762db2be94e68100b267611327378a1e94a8fbe0cc1666538adL106), I'm going to debug why a little further